src/test/resources/conversions/v10/DecimalDirections-results.json

```diff
@@ -26180,6 +26180,7 @@
     }
   ],
   "id": "650366caf910740b202baa4d",
+  "version": 10,
```

Comment:
why add version to a pre-v12 file?

Reply:
because this changeset has a validation test. Even though this is a v10 changeset, it always tries to validate against the current schema